### PR TITLE
fix crash when config is not properly initialized

### LINKED ISSUE #12709: panic: runtime error: invalid memory address or nil pointer dereference
Starting the MinIO server fails with a SIGSEGV error
## Expected Behavior

It should just start 😄

## Current Behavior

It doesn't work 😞 
(I replaced some sensitive values with `nope`)
```
./minio server /mnt/data

API: SYSTEM()
Time: 09:52:12 UTC 07/14/2021
DeploymentID: aaa0ceb9-ff2d-43c0-94cd-be8fd8e99624
Error: Unable to initialize config, some features may be missing open config/config.json: not a directory (*fmt.wrapError)
       3: cmd/server-main.go:394:cmd.initAllSubsystems()
       2: cmd/server-main.go:325:cmd.initServer()
       1: cmd/server-main.go:545:cmd.serverMain()
API: http://nope:9000  http://127.0.0.1:9000
RootUser: nope
RootPass: nope

Console: http://nope:37017 http://127.0.0.1:37017
RootUser: nope
RootPass: nope

Command-line: https://docs.min.io/docs/minio-client-quickstart-guide
   $ mc alias set myminio http://nope:9000 nope nope

Documentation: https://docs.min.io

WARNING: Console endpoint is listening on a dynamic port (37017), please use --console-address ":PORT" to choose a static port.
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xd83ce6]

goroutine 73 [running]:
sync.(*Mutex).Lock(...)
        sync/mutex.go:74
github.com/minio/minio/internal/config/identity/openid.(*Config).ProviderEnabled(0x6280aa0, 0x49e2700)
        github.com/minio/minio/internal/config/identity/openid/jwt.go:105 +0x46
github.com/minio/minio/cmd.(*IAMSys).Init(0xc0003087e0, 0x49e27f8, 0xc0003f7ac0, 0x4a2ffa8, 0xc000265d50)
        github.com/minio/minio/cmd/iam.go:653 +0x8ef
created by github.com/minio/minio/cmd.serverMain
        github.com/minio/minio/cmd/server-main.go:562 +0xa3f
```

## Possible Solution

no idea

## Steps to Reproduce (for bugs)

```bash
ufw enable
ufw allow 9000
wget https://dl.min.io/server/minio/release/linux-amd64/minio
chmod +x minio
echo "MINIO_ROOT_USER=nope MINIO_ROOT_PASSWORD=nope ./minio server /mnt/data" > minio.sh
chmod +x minio.sh
./minio.sh
```

## Context

I was just trying to start the MinIO server, commands above

## Regression

I don't know

## Your Environment

* Version used (`minio --version`): `minio version RELEASE.2021-07-12T02-44-53Z`
* Server setup and configuration: Linode Ubuntu 20.04 LTS server with ufw enabled
* Operating System and version (`uname -a`): `Linux localhost 5.4.0-72-generic #80-Ubuntu SMP Mon Apr 12 17:35:00 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux` (Ubuntu 20.04 LTS)

### PR DESCRIPTION
**Pull request by harshavardhana**
## Description
fix crash when config is not properly initialized

## Motivation and Context
fixes #12709

## How to test this PR?
Build a container image locally 
```
~ TAG=minio/minio:dev make docker
```

```yaml
version: '3.7'  
                                                                                                                                     
# starts 4 docker containers running minio server instances.
# using nginx reverse proxy, load balancing, you can access                                              
# it through port 9000.                                                  
services:                                                                                                    
  minio1:                                                 
    image: minio/minio:dev                                      
    hostname: minio1                                     
    volumes:                                                                 
      - data1-1:/data1
      - data1-2:/data2                                                      
    expose:        
      - "9000"    
      - "9001"  
    environment:
      MINIO_ROOT_USER: minio
      MINIO_ROOT_PASSWORD: minio123
      MINIO_IDENTITY_OPENID_CONFIG_URL: "https://alevsk-minio.auth0.com/.well-known/openid-configuration"
      MINIO_IDENTITY_OPENID_CLIENT_ID: "UOhR2Gw2EuPbSjFla96ppLHWEypzZUh2"
      MINIO_IDENTITY_OPENID_CLIENT_SECRET: "cB9_oc1Mx__cNFDdmKmhjNC18O5Udtvp8T0p94Ug8FCM8-PUB4OgFqPXScj01mLm"
      MINIO_IDENTITY_OPENID_SCOPES: "openid,profile,email"
      MINIO_IDENTITY_OPENID_CLAIM_NAME: "https://min.io/policy"
      MINIO_BROWSER_REDIRECT_URL: "http://10.0.0.67:9001"
    command: server --console-address ":9001" http://minio{1...4}/data{1...2}
    healthcheck:
      test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
      interval: 30s
      timeout: 20s
      retries: 3
                                                                                                                                     
  minio2:                                                      
    image: minio/minio:dev
    hostname: minio2
    volumes:
      - data2-1:/data1
      - data2-2:/data2
    expose:
      - "9000"
      - "9001"
    environment:
      MINIO_ROOT_USER: minio                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
      MINIO_ROOT_PASSWORD: minio123                      
      MINIO_IDENTITY_OPENID_CONFIG_URL: "https://alevsk-minio.auth0.com/.well-known/openid-configuration"
      MINIO_IDENTITY_OPENID_CLIENT_ID: "UOhR2Gw2EuPbSjFla96ppLHWEypzZUh2"
      MINIO_IDENTITY_OPENID_CLIENT_SECRET: "cB9_oc1Mx__cNFDdmKmhjNC18O5Udtvp8T0p94Ug8FCM8-PUB4OgFqPXScj01mLm"
      MINIO_IDENTITY_OPENID_SCOPES: "openid,profile,email"
      MINIO_IDENTITY_OPENID_CLAIM_NAME: "https://min.io/policy"
      MINIO_BROWSER_REDIRECT_URL: "http://10.0.0.67:9001"
    command: server --console-address ":9001" http://minio{1...4}/data{1...2}
    healthcheck:
      test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
      interval: 30s
      timeout: 20s
      retries: 3

  minio3:
    image: minio/minio:dev
    hostname: minio3
    volumes:
      - data3-1:/data1
      - data3-2:/data2
    expose:
      - "9000"
      - "9001"
    environment:
      MINIO_ROOT_USER: minio
      MINIO_ROOT_PASSWORD: minio123
      MINIO_IDENTITY_OPENID_CONFIG_URL: "https://alevsk-minio.auth0.com/.well-known/openid-configuration"
      MINIO_IDENTITY_OPENID_CLIENT_ID: "UOhR2Gw2EuPbSjFla96ppLHWEypzZUh2"
      MINIO_IDENTITY_OPENID_CLIENT_SECRET: "cB9_oc1Mx__cNFDdmKmhjNC18O5Udtvp8T0p94Ug8FCM8-PUB4OgFqPXScj01mLm"
      MINIO_IDENTITY_OPENID_SCOPES: "openid,profile,email"
      MINIO_IDENTITY_OPENID_CLAIM_NAME: "https://min.io/policy"
      MINIO_BROWSER_REDIRECT_URL: "http://10.0.0.67:9001"
    command: server --console-address ":9001" http://minio{1...4}/data{1...2}
    healthcheck:
      test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
      interval: 30s
      timeout: 20s
      retries: 3

  minio4:
    image: minio/minio:dev
    hostname: minio4
    volumes:
      - data4-1:/data1
      - data4-2:/data2
    expose:
      - "9000"
      - "9001"
    environment:
      MINIO_ROOT_USER: minio
      MINIO_ROOT_PASSWORD: minio123
      MINIO_IDENTITY_OPENID_CONFIG_URL: "https://alevsk-minio.auth0.com/.well-known/openid-configuration"
      MINIO_IDENTITY_OPENID_CLIENT_ID: "UOhR2Gw2EuPbSjFla96ppLHWEypzZUh2"
      MINIO_IDENTITY_OPENID_CLIENT_SECRET: "cB9_oc1Mx__cNFDdmKmhjNC18O5Udtvp8T0p94Ug8FCM8-PUB4OgFqPXScj01mLm"
      MINIO_IDENTITY_OPENID_SCOPES: "openid,profile,email"
      MINIO_IDENTITY_OPENID_CLAIM_NAME: "https://min.io/policy"
      MINIO_BROWSER_REDIRECT_URL: "http://10.0.0.67:9001"
    command: server --console-address ":9001" http://minio{1...4}/data{1...2}
    healthcheck:
      test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
      interval: 30s
      timeout: 20s
      retries: 3

  nginx:
    image: nginx:1.19.2-alpine
    hostname: nginx
    volumes:
      - ./nginx.conf:/etc/nginx/nginx.conf:ro
    ports:
      - "9000:9000"
      - "9001:9001"
    depends_on:
      - minio1
      - minio2
      - minio3
      - minio4

## By default this config uses default local driver,
## For custom volumes replace with volume driver configuration.
volumes:
  data1-1:
  data1-2:
  data2-1:
  data2-2:
  data3-1:
  data3-2:
  data4-1:
  data4-2:
```

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Documentation updated
- [ ] Unit tests added/updated

### CODE REVIEW

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-zoned.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-zoned.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-zoned.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

**Review by poornas — APPROVED** [review]

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-zoned.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-zoned.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-zoned.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

File: internal/config/identity/openid/jwt_test.go [hunk 1]
```diff
@@ -138,21 +138,21 @@ func TestJWT(t *testing.T) {
...
+	cfg := Config{
+		RWMutex: &sync.RWMutex{},
+	}
 	cfg.JWKS.URL = u1
 	cfg.publicKeys = keys
-	jwt := NewJWT(cfg)
-	if jwt.ID() != "jwt" {
-		t.Fatalf("Uexpected id %s for the validator", jwt.ID())
+	if cfg.ID() != "jwt" {
+		t.Fatalf("Uexpected id %s for the validator", cfg.ID())
```
Comment (krisis) [inline internal/config/identity/openid/jwt_test.go]: typo
```suggestion
		t.Fatalf("Unexpected id %s for the validator", cfg.ID())
```

File: internal/config/identity/openid/jwt_test.go [hunk 2]
```diff
@@ -90,16 +90,16 @@ func TestJWTAzureFail(t *testing.T) {
...
```
Comment (krisis) [inline internal/config/identity/openid/jwt_test.go]:
```suggestion
		t.Fatalf("Unexpected id %s for the validator", cfg.ID())
```

File: internal/config/identity/openid/validators_test.go [hunk 3]
```diff
@@ -95,7 +95,7 @@ func TestValidators(t *testing.T) {
 
 	cfg := Config{}
 	cfg.JWKS.URL = u
-	if err = vrs.Add(NewJWT(cfg)); err != nil {
+	if err = vrs.Add(&cfg); err != nil {
```
Comment (krisis) [inline internal/config/identity/openid/validators_test.go:98]:
```suggestion
        cfg.Enabled = true
	if err = vrs.Add(&cfg); err != nil {
```

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-zoned.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

**Review by krisis — APPROVED** [review]

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-zoned.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-zoned.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-zoned.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

**Comment by minio-trusted** [comment]
```
Error running mint automation
+ mkdir -p 12714-e3685a0/gopath/src/github.com/minio
+ git -C 12714-e3685a0/gopath/src/github.com/minio clone --quiet https://github.com/minio/minio.git
+ git -C 12714-e3685a0/gopath/src/github.com/minio/minio remote add minio https://github.com/minio/minio.git
+ git -C 12714-e3685a0/gopath/src/github.com/minio/minio fetch --quiet minio pull/12714/head:pr12714
+ git -C 12714-e3685a0/gopath/src/github.com/minio/minio checkout --quiet pr12714
+ GO111MODULE=on
+ GOPATH=/home/mint-auto/mint-auto/12714-e3685a0/gopath
+ make -C 12714-e3685a0/gopath/src/github.com/minio/minio --quiet
go: downloading github.com/minio/madmin-go v1.0.16-0.20210713203601-e2a5b7c4d8ae
go: downloading github.com/minio/cli v1.22.0
go: downloading github.com/dustin/go-humanize v1.0.0
go: downloading github.com/Azure/azure-storage-blob-go v0.10.0
go: downloading github.com/json-iterator/go v1.1.11
go: downloading cloud.google.com/go v0.65.0
go: downloading google.golang.org/api v0.31.0
go: downloading github.com/minio/pkg v1.0.10
go: downloading github.com/jcmturner/gokrb5/v8 v8.4.2
go: downloading github.com/minio/minio-go/v7 v7.0.13-0.20210706013812-337aa536abe2
go: downloading github.com/colinmarc/hdfs/v2 v2.2.0
go: downloading github.com/minio/highwayhash v1.0.2
go: downloading github.com/fatih/color v1.12.0
go: downloading github.com/Azure/azure-pipeline-go v0.2.2
go: downloading golang.org/x/sys v0.0.0-20210510120138-977fb7262007
go: downloading github.com/Shopify/sarama v1.27.2
go: downloading github.com/minio/sio v0.3.0
go: downloading github.com/secure-io/sio-go v0.3.1
go: downloading github.com/valyala/tcplisten v0.0.0-20161114210144-ceec8f93295a
go: downloading github.com/minio/sha256-simd v1.0.0
go: downloading github.com/xdg/scram v0.0.0-20180814205039-7eeb5667e42c
go: downloading github.com/golang-jwt/jwt v3.2.1+incompatible
go: downloading github.com/minio/kes v0.14.0
go: downloading golang.org/x/crypto v0.0.0-20210421170649-83a5a9bb288b
go: downloading github.com/VividCortex/ewma v1.1.1
go: downloading github.com/bits-and-blooms/bloom/v3 v3.0.1
go: downloading github.com/go-openapi/loads v0.20.2
go: downloading github.com/klauspost/compress v1.13.1
go: downloading github.com/miekg/dns v1.1.35
go: downloading github.com/klauspost/readahead v1.3.1
go: downloading github.com/dchest/siphash v1.2.1
go: downloading github.com/bcicen/jstream v1.0.1
go: downloading github.com/klauspost/pgzip v1.2.5
go: downloading github.com/djherbis/atime v1.0.0
go: downloading github.com/gorilla/mux v1.8.0
go: downloading github.com/cheggaaa/pb v1.0.29
go: downloading github.com/google/uuid v1.1.2
go: downloading github.com/minio/console v0.7.6-0.20210712213110-54dfcbd378d2
go: downloading github.com/cespare/xxhash v1.1.0
go: downloading github.com/klauspost/reedsolomon v1.9.11
go: downloading github.com/minio/rpc v1.0.0
go: downloading github.com/minio/selfupdate v0.3.1
go: downloading github.com/mitchellh/go-homedir v1.1.0
go: downloading github.com/philhofer/fwd v1.1.1
go: downloading github.com/minio/zipindex v0.2.0
go: downloading github.com/pierrec/lz4 v2.6.0+incompatible
go: downloading github.com/prometheus/client_golang v1.8.0
go: downloading github.com/prometheus/procfs v0.6.0
go: downloading github.com/prometheus/client_model v0.2.0
go: downloading github.com/rs/cors v1.7.0
go: downloading github.com/tinylib/msgp v1.1.6-0.20210521143832-0becd170c402
go: downloading github.com/valyala/bytebufferpool v1.0.0
go: downloading go.etcd.io/etcd v0.5.0-alpha.5.0.20200910180754-dd1b699fc489
go: downloading go.uber.org/atomic v1.7.0
go: downloading github.com/cespare/xxhash/v2 v2.1.1
go: downloading github.com/beevik/ntp v0.3.0
go: downloading github.com/coredns/coredns v1.4.0
go: downloading github.com/go-ldap/ldap v3.0.2+incompatible
go: downloading go.uber.org/zap v1.16.1-0.20210329175301-c23abee72d19
go: downloading github.com/minio/simdjson-go v0.2.1
go: downloading github.com/eclipse/paho.mqtt.golang v1.3.0
go: downloading github.com/elastic/go-elasticsearch/v7 v7.12.0
go: downloading github.com/go-sql-driver/mysql v1.5.0
go: downloading github.com/gomodule/redigo v2.0.0+incompatible
go: downloading github.com/lib/pq v1.9.0
go: downloading github.com/nats-io/nats.go v1.10.0
go: downloading github.com/nats-io/stan.go v0.8.3
go: downloading github.com/nsqio/go-nsq v1.0.8
go: downloading github.com/olivere/elastic/v7 v7.0.22
go: downloading github.com/pkg/errors v0.9.1
go: downloading github.com/streadway/amqp v1.0.0
go: downloading github.com/montanaflynn/stats v0.6.6
go: downloading github.com/ncw/directio v1.0.5
go: downloading golang.org/x/time v0.0.0-20210220033141-f8bda1e9f3ba
go: downloading github.com/mattn/go-colorable v0.1.8
go: downloading github.com/mattn/go-isatty v0.0.13
go: downloading github.com/bits-and-blooms/bitset v1.2.0
go: downloading github.com/golang/protobuf v1.5.2
go: downloading github.com/minio/csvparser v1.0.0
go: downloading github.com/go-ldap/ldap/v3 v3.2.4
go: downloading github.com/minio/parquet-go v1.0.0
go: downloading github.com/alecthomas/participle v0.2.1
go: downloading github.com/mattn/go-runewidth v0.0.13
go: downloading github.com/mattn/go-ieproxy v0.0.1
go: downloading github.com/xdg/stringprep v1.0.0
go: downloading golang.org/x/net v0.0.0-20210421230115-4e50805a0758
go: downloading github.com/minio/argon2 v1.0.0
go: downloading github.com/shirou/gopsutil v0.0.0-20190901111213-e4ec7b275ada
go: downloading github.com/nats-io/nuid v1.0.1
go: downloading github.com/golang/snappy v0.0.3
go: downloading github.com/klauspost/cpuid v1.3.1
go: downloading github.com/prometheus/common v0.14.0
go: downloading github.com/go-openapi/analysis v0.20.0
go: downloading github.com/go-openapi/spec v0.20.3
go: downloading github.com/go-openapi/swag v0.19.14
go: downloading github.com/rivo/uniseg v0.2.0
go: downloading github.com/Azure/go-ntlmssp v0.0.0-20200615164410-66371956d46c
go: downloading github.com/go-asn1-ber/asn1-ber v1.5.1
go: downloading github.com/gogo/protobuf v1.3.2
go: downloading github.com/gorilla/websocket v1.4.2
go: downloading github.com/beorn7/perks v1.0.1
go: downloading github.com/lestrrat-go/jwx v1.2.0
go: downloading github.com/rjeczalik/notify v0.9.2
go: downloading github.com/fatih/structs v1.1.0
go: downloading gopkg.in/yaml.v2 v2.4.0
go: downloading github.com/nats-io/jwt v1.1.0
go: downloading github.com/nats-io/nkeys v0.2.0
go: downloading golang.org/x/text v0.3.6
go: downloading github.com/jcmturner/dnsutils/v2 v2.0.0
go: downloading github.com/jcmturner/gofork v1.0.0
go: downloading github.com/hashicorp/go-uuid v1.0.2
go: downloading github.com/jcmturner/aescts/v2 v2.0.0
go: downloading github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: downloading github.com/modern-go/reflect2 v1.0.1
go: downloading git.apache.org/thrift.git v0.13.0
go: downloading github.com/jcmturner/rpc/v2 v2.0.3
go: downloading google.golang.org/protobuf v1.26.0
go: downloading github.com/tidwall/gjson v1.7.5
go: downloading github.com/tidwall/sjson v1.1.6
go: downloading go.uber.org/multierr v1.6.0
go: downloading gopkg.in/ini.v1 v1.57.0
go: downloading github.com/minio/md5-simd v1.1.1
go: downloading github.com/rs/xid v1.2.1
go: downloading github.com/mailru/easyjson v0.7.6
go: downloading github.com/tidwall/pretty v1.1.0
go: downloading github.com/tidwall/match v1.0.3
go: downloading github.com/shirou/gopsutil/v3 v3.21.6
go: downloading github.com/matttproud/golang_protobuf_extensions v1.0.2-0.20181231171920-c182affec369
go: downloading github.com/davecgh/go-spew v1.1.1
go: downloading gopkg.in/jcmturner/gokrb5.v7 v7.5.0
go: downloading github.com/eapache/queue v1.1.0
go: downloading github.com/eapache/go-resiliency v1.2.0
go: downloading github.com/eapache/go-xerial-snappy v0.0.0-20180814174437-776d5712da21
go: downloading github.com/rcrowley/go-metrics v0.0.0-20200313005456-10cdbea86bc0
go: downloading github.com/jcmturner/goidentity/v6 v6.0.1
go: downloading github.com/josharian/intern v1.0.0
go: downloading github.com/klauspost/cpuid/v2 v2.0.4
go: downloading github.com/go-openapi/strfmt v0.20.0
go: downloading github.com/go-openapi/jsonpointer v0.19.5
go: downloading github.com/lestrrat-go/option v1.0.0
go: downloading github.com/lestrrat-go/iter v1.0.1
go: downloading github.com/lestrrat-go/httpcc v1.0.0
go: downloading github.com/lestrrat-go/backoff/v2 v2.0.7
go: downloading github.com/lestrrat-go/blackmagic v1.0.0
go: downloading github.com/asaskevich/govalidator v0.0.0-20200907205600-7a23bdc65eef
go: downloading github.com/mitchellh/mapstructure v1.4.1
go: downloading github.com/go-openapi/errors v0.19.9
go: downloading go.mongodb.org/mongo-driver v1.4.6
go: downloading cloud.google.com/go/storage v1.10.0
go: downloading github.com/go-openapi/jsonreference v0.19.5
go: downloading github.com/tklauser/go-sysconf v0.3.6
go: downloading github.com/PuerkitoBio/purell v1.1.1
go: downloading github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578
go: downloading github.com/tklauser/numcpus v0.2.2
go: downloading google.golang.org/genproto v0.0.0-20201110150050-8816d57aaa9a
go: downloading github.com/googleapis/gax-go v2.0.2+incompatible
go: downloading google.golang.org/grpc v1.37.0
go: downloading go.opencensus.io v0.22.5
go: downloading github.com/googleapis/gax-go/v2 v2.0.5
go: downloading github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e
go: downloading github.com/go-stack/stack v1.8.0
go: downloading go.etcd.io/etcd/api/v3 v3.5.0-beta.4
go: downloading go.etcd.io/etcd/client/v3 v3.5.0-beta.4
go: downloading go.etcd.io/etcd/client/pkg/v3 v3.5.0-beta.4
go: downloading github.com/coreos/go-semver v0.3.0
go: downloading github.com/coreos/go-systemd v0.0.0-20190719114852-fd7a80b32e1f
go: downloading github.com/go-openapi/runtime v0.19.24
go: downloading github.com/minio/direct-csi v1.3.5-0.20210601185811-f7776f7961bf
go: downloading github.com/minio/mc v0.0.0-20210626002108-cebf3318546f
go: downloading github.com/minio/operator v0.0.0-20210616045941-65f31f5f78ae
go: downloading github.com/unrolled/secure v1.0.7
go: downloading k8s.io/client-go v0.21.1
go: downloading k8s.io/api v0.21.1
go: downloading k8s.io/apimachinery v0.21.1
go: downloading github.com/jessevdk/go-flags v1.4.0
go: downloading golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
go: downloading github.com/coreos/go-oidc v2.2.1+incompatible
go: downloading github.com/go-openapi/validate v0.20.2
go: downloading gopkg.in/square/go-jose.v2 v2.3.1
go: downloading github.com/pquerna/cachecontrol v0.0.0-20180517163645-1555304b9b35
go: downloading github.com/coreos/go-systemd/v22 v22.3.1
go: downloading github.com/docker/go-units v0.4.0
go: downloading k8s.io/kube-openapi v0.0.0-20210305001622-591a79e4bda7
go: downloading github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
go: downloading github.com/mb0/glob v0.0.0-20160210091149-1eb79d2de6c4
go: downloading github.com/minio/colorjson v1.0.1
go: downloading github.com/posener/complete v1.2.3
go: downloading github.com/inconshreveable/go-update v0.0.0-20160112193335-8152e7eb6ccf
go: downloading maze.io/x/duration v0.0.0-20160924141736-faac084b6075
go: downloading gopkg.in/h2non/filetype.v1 v1.0.5
go: downloading github.com/pkg/xattr v0.4.1
go: downloading github.com/pkg/profile v1.3.0
go: downloading github.com/dgrijalva/jwt-go v3.2.0+incompatible
go: downloading sigs.k8s.io/controller-runtime v0.8.0
go: downloading github.com/minio/filepath v1.0.0
go: downloading github.com/minio/operator/logsearchapi v0.0.0-20210604224119-7e256f98cf90
go: downloading k8s.io/klog v1.0.0
go: downloading golang.org/x/term v0.0.0-20210406210042-72f3dc4e9b72
go: downloading k8s.io/klog/v2 v2.8.0
go: downloading github.com/hashicorp/go-multierror v1.0.0
go: downloading github.com/georgysavva/scany v0.2.7
go: downloading github.com/google/gofuzz v1.1.0
go: downloading gopkg.in/inf.v0 v0.9.1
go: downloading sigs.k8s.io/yaml v1.2.0
go: downloading sigs.k8s.io/structured-merge-diff v1.0.1-0.20191108220359-b1b620dd3f06
go: downloading github.com/google/go-cmp v0.5.5
go: downloading github.com/hashicorp/errwrap v1.0.0
go: downloading github.com/stretchr/testify v1.7.0
go: downloading github.com/go-logr/logr v0.4.0
go: downloading github.com/stretchr/objx v0.2.0
go: downloading github.com/pmezard/go-difflib v1.0.1-0.20181226105442-5d4384ee4fb2
go: downloading gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
go: downloading sigs.k8s.io/structured-merge-diff/v4 v4.1.0
go: downloading github.com/emicklei/go-restful v2.9.5+incompatible
go: downloading k8s.io/utils v0.0.0-20201110183641-67b214c5f920
go: downloading github.com/googleapis/gnostic v0.5.1
go: downloading github.com/hashicorp/golang-lru v0.5.4
go: downloading gopkg.in/jcmturner/dnsutils.v1 v1.0.1
go: downloading gopkg.in/jcmturner/aescts.v1 v1.0.1
go: downloading gopkg.in/jcmturner/rpc.v1 v1.1.0
+ podman build --quiet --tag docker.io/minio/minio:12714-e3685a0 --file 12714-e3685a0/gopath/src/github.com/minio/minio/Dockerfile.dev 12714-e3685a0/gopath/src/github.com/minio/minio
+ podman push --quiet docker.io/minio/minio:12714-e3685a0

```

**Pull request merged**